### DISCUSSION TITLE
@prisma/client did not initialize yet

### DISCUSSION THREAD

**Discussion by dsignzmedia** [architect/architect#1376] [Q&A]
Hi,
We are developing an application using Remix and Architect. We want to deploy this application to AWS Lambda. Our application's database is Postgresql and we are using Prisma for database operations.

Using Architect, we were able to deploy the app to AWS Lambda. However, we get this error 

"errorType": "Error",
    "errorMessage": "@prisma/client did not initialize yet. Please run \"prisma generate\" and try to import it again.\nIn case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues",
    "stack": [
        "Error: @prisma/client did not initialize yet. Please run \"prisma generate\" and try to import it again.",
        "In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues",
        "    at new PrismaClient (/var/task/node_modules/.prisma/client/index.js:3:11)",

Any inputs on this would be really helpful.

Also, we are using AWS RDS for Postgresql, and we have given the connection string as DATABASE_URL environment variables. 

I tried to used plugins to include prisma/client, how I am not sure how to provide the prisma generate command in .arc file / package.json.

**Comment by michaelwills** [comment]
@dsignzmedia did you find a resolution for this?

**Reply by danactive** [reply]
Workaround documented in related [Issue](https://github.com/architect/architect/issues/1377)
